scalableminds/webknossos
# Make copy-to-my-account button an async button

### URL of deployed dev instance (used for testing):
- https://disablecopybutton.webknossos.xyz

### Steps to test:
- open a tracing from user A while being logged in as user B
- click copy-to-my-account button
- button should be disabled as long as the request is running

### Issues:
-  fixes #3266

------
